crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -169,6 +172,15 @@ pub struct BuildSpecCommand {
 	/// Whether the genesis code file should be generated.
 	#[arg(short = 'C', long = "genesis-code")]
 	pub(crate) genesis_code: bool,
+	/// Whether to build the Wasm runtime deterministically.
```

Comment:
```suggestion
	/// Whether to build the runtime deterministically.
```
Expected to be PVM one day, no need to be explicit now. There is also no mention that it requires docker/podman etc

Reply:
https://github.com/r0gue-io/pop-cli/pull/425/commits/e1d47d067f6c7c5bcd84bdc714601f0d5c91a06c